script_facade/client/client_106.py

```diff
@@ -103,3 +119,20 @@ def rx_history_query(patient_fname, patient_lname, patient_dob):
 
     meds = parse_rx_history_response(xml_body)
     return meds
+
+
+def patient_lookup_query(patient_fname, patient_lname, patient_dob):
```

Comment:
seems excessive to have `patient_lookup_query` prepend each arg w/ `patient_`...